2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Configure stats pid to 30942
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Loading settings from /Users/samuel.shapley/.config/wandb/settings
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Loading settings from /Users/samuel.shapley/projects/openai-launchpad/openai-launchpad/wandb/settings
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:_log_setup():524] Logging user logs to /Users/samuel.shapley/projects/openai-launchpad/openai-launchpad/wandb/run-20240105_210201-31nn601g/logs/debug.log
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/samuel.shapley/projects/openai-launchpad/openai-launchpad/wandb/run-20240105_210201-31nn601g/logs/debug-internal.log
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x13246f8d0>
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:init():564] calling init triggers
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:init():614] starting backend
2024-01-05 21:02:01,344 INFO    MainThread:30942 [wandb_init.py:init():618] setting up manager
2024-01-05 21:02:01,345 INFO    MainThread:30942 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-01-05 21:02:01,345 INFO    MainThread:30942 [wandb_init.py:init():624] backend started and connected
2024-01-05 21:02:01,346 INFO    MainThread:30942 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-05 21:02:01,346 INFO    MainThread:30942 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-05 21:02:01,347 INFO    MainThread:30942 [wandb_init.py:init():716] updated telemetry
2024-01-05 21:02:01,376 INFO    MainThread:30942 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-01-05 21:02:01,797 INFO    MainThread:30942 [wandb_run.py:_on_init():2254] communicating current version
2024-01-05 21:02:01,878 INFO    MainThread:30942 [wandb_run.py:_on_init():2263] got version response 
2024-01-05 21:02:01,878 INFO    MainThread:30942 [wandb_init.py:init():800] starting run threads in backend
2024-01-05 21:02:01,940 INFO    MainThread:30942 [wandb_run.py:_console_start():2233] atexit reg
2024-01-05 21:02:01,940 INFO    MainThread:30942 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-05 21:02:01,940 INFO    MainThread:30942 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-05 21:02:01,940 INFO    MainThread:30942 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-05 21:02:01,940 INFO    MainThread:30942 [wandb_init.py:init():841] run started, returning control to user process
2024-01-05 21:02:16,917 INFO    MainThread:30942 [wandb_run.py:_finish():1962] finishing run samuel-shapley/openai-launchpad/31nn601g
2024-01-05 21:02:16,917 INFO    MainThread:30942 [jupyter.py:save_history():445] not saving jupyter history
2024-01-05 21:02:16,917 INFO    MainThread:30942 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-05 21:02:16,917 INFO    MainThread:30942 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-01-05 21:02:16,917 INFO    MainThread:30942 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-05 21:02:16,918 INFO    MainThread:30942 [wandb_run.py:_restore():2185] restore
2024-01-05 21:02:16,918 INFO    MainThread:30942 [wandb_run.py:_restore():2191] restore done
2024-01-05 21:02:21,014 INFO    MainThread:30942 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-01-05 21:02:21,014 INFO    MainThread:30942 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-01-05 21:02:21,022 INFO    MainThread:30942 [wandb_run.py:_footer_sync_info():3796] logging synced files
